Resources/Prototypes/Traits/mental.yml

```diff
@@ -28,13 +28,11 @@
       species:
         - Oni
         - Shadowkin
-    - !type:CharacterTraitRequirement
-      inverted: true
-      traits:
```

Comment:
why would you remove trait requirement here? This commentary applies to the whole file.